myuw/templates/handlebars/card/accounts/library.html

```diff
@@ -59,6 +59,8 @@ <h4 class="card-badge-label">You owe</h4>
                 <span class="btn btn-default card-action-btn pull-right"><a href="http://search.lib.uw.edu/account" target="_blank" data-linklabel="Your Library Account">Access library account</a></span>
             </div>
         </div>
-
+    {{else}}
+    <div><p class="text-danger"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> An error occurred and MyUW cannot load your library account information right now. In the meantime, try the <a href="http://www.lib.washington.edu/" data-linklabel="UW Libraries" style="text-decoration:underline;" target="_blank">UW Libraries page</a>.</p></div>
```

Comment:
I recommend removing the inline styles here for underlining and adding that style to the CSS.

Reply:
removed